src/rbd.cc

```diff
@@ -3391,13 +3401,6 @@ if (!set_conf_param(v, p1, p2, p3)) { \
 	 << std::endl;
     return EXIT_FAILURE;
   }
-  if ((opt_cmd == OPT_SNAP_CREATE || opt_cmd == OPT_SNAP_ROLLBACK ||
```

Comment:
Why remove this single check just so you can re-add it for each individual command below?